classes/SpecificPrice.php

```diff
@@ -467,12 +472,13 @@ protected static function formatIntInQuery($first_value, $second_value)
      * @param int  $id_customer
      * @param int  $id_cart
      * @param int  $real_quantity
+     * @param int  $id_zone
      *
      * @return array
      */
     public static function getSpecificPrice($id_product, $id_shop, $id_currency, $id_country, $id_group, $quantity,
                                             $id_product_attribute = null, $id_customer = 0, $id_cart = 0,
-                                            $real_quantity = 0)
+                                            $real_quantity = 0, $id_zone)
```

Comment:
default value missing here

Reply:
In other places too. Cf. Codacy report.

Reply:
Should I change the others getSpecificPrice's parameters?